libs/full/parcelport_lci/include/hpx/parcelport_lci/receiver.hpp

```diff
@@ -46,54 +90,157 @@ namespace hpx::parcelset::policies::lci {
         bool background_work() noexcept
         {
             // We first try to accept a new connection
-            connection_ptr connection = accept();
+            LCI_request_t request = accept();
```

Comment:
Wrapping the `LCI_request_t` into a RAII type that releases it at destruction time would improve exception safety.